great-idea/css/index.css

```diff
@@ -63,4 +63,168 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+
+header {
+	height: 70px;
+}
+
+nav{
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+}
+
+a {
+	margin: 0 10px;
+  	padding: 20px;
+  	border-radius: 20px;
+  	color: gray;
+	margin-bottom: 20px;
+	text-decoration: none;
+	
+}	
+
+.logo{
+	position: absolute;
+	right: 50px;
+	top: 30px;
+}
+
+.head{
+  	padding: 50px;
+	width: 49%;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+
+}
+
+.first-box{
+	display: flex;
+	flex-wrap: wrap;
+}
+
+
+h1{
+	font-size: 70px; 
+	text-align: center;
+	margin-bottom: 50px;
+}
+
+.call-to-action{
+	border: 1px solid black;
+	padding: 10px 30px;
+}
+
+.rounded{
+	border-radius: 50%;
+}
+
+.middle{
+	border-top: 1px solid black;
+	border-bottom: 1px solid black;
+	border-width: 70%;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	
+}
+
+.two-columns{
+  padding: 30px;
+  width: 45%;
+  text-align: justify;
+}
+
+.middle-img{
+	width: 90%;
+}
+
+.three-columns{
+	padding: 30px;
+	width: 30%;
+}
+
+.contact .postal{
+	line-height: 1;
+}
+.contact{
+	padding-left: 20px;
+	line-height: 2;
+	border-top: 0;
+}
+
+footer{
+	padding: 20px 0;
+	text-align: center;
+}
+
+
+/*----Service page----*/
+
+body{
+	width: 80%;
+	padding: 0 5% 0 5%;
+}
+
+.top-section{
+	display:flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: center;
+	width: 95%;
+
+}
+
+.top-section-text{
+	padding: 20px;
+
+}
+
+
+.services-op{
+	display:flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: center;
+	border-top: 1px solid black;
```

Comment:
You need to use something like `<hr>` . So, you can easily align them with other elements. Give it a separate width if you need to.  [Horizontal Ruler](https://www.w3schools.com/tags/tag_hr.asp)